ui/app/serializers/clients/config.js

```diff
@@ -5,24 +5,27 @@
 
 import ApplicationSerializer from '../application';
 
-export default ApplicationSerializer.extend({
+export default class ClientsConfigSerializer extends ApplicationSerializer {
+  // these attrs are readOnly
+  attrs = {
+    billingStartTimestamp: { serialize: false },
+    minimumRetentionMonths: { serialize: false },
+    reportingEnabled: { serialize: false },
+  };
+
   normalizeResponse(store, primaryModelClass, payload, id, requestType) {
-    if (!payload.data) {
```

Comment:
@chelshaw I suspect this safe to remove, but thought I should check with you first? It was added 3ish years ago.